src/user/types/user.types.ts

```diff
@@ -0,0 +1,3 @@
+// import { UserEntity } from "../entities/user.entity";
+
+// export type UserType = Omit<UserEntity, 'hashPassword'>;
```

Comment:
Remove the commented code or the entire file

Reply:
✅